crates/solidity/testing/perf/benches/iai/dataset.rs

```diff
@@ -243,3 +246,55 @@ pub fn run_query(trees: &Vec<Node>) {
         "Function names don't match: {results:#?}"
     );
 }
+
+struct NoOpResolver;
+
+impl PathResolver for NoOpResolver {
+    fn resolve_path(&self, _context_path: &str, path_to_resolve: &str) -> Option<String> {
+        Some(path_to_resolve.to_string())
+    }
+}
+
+pub fn run_create_bindings() {
```

Comment:
Based on our previous discussion, creating built-ins will be handled by the parent `Compilation`, so if you agree, let's skip this from perf for now.
WDYT of having two tests for now?

- `build_definitions`
- `resolve_references`

Suggested this naming to disambiguate the actual functionality we are testing the final perf dashboard.

Reply:
Removing the test sounds good, I'll make that change as well as the renames.

The `setup` attribute of `library_benchmark` doesn't seem to accept arbitrary expressions, only function names. I will double check to see if there's a way around it.

Reply:
Ok, `setup` doesn't work, but there is an alternative: a bench can specify an `args` parameter that *does* allow arbitrary expressions.

Reply:
Sorry about that. In that case, `#[bench]` wouldn't produce the same test name, thus resulting in mangled IDs in the dashboard.
To maintain this correctly, we need to separate them into modules with proper `setup()` and `run()` functions.
I took a stab at it to make sure my suggestion makes sense, and submitted a PR against your branch:
https://github.com/manastech/slang/pull/1

Reply:
Ah, I see. I saw that the test name was larger but didn't think it'd be a problem, probably because it was truncated in the CLI output.

And now that I think of it, I observed that the test program was doing a huge spike in memory usage right before the last test, up to the point where it would be killed by the kernel. It was probably the harness trying to serialize the parameters of the test case :facepalm: